views/movies/edit.hbs

```diff
@@ -0,0 +1,18 @@
+<h2>Edit Movie</h2>
+
+{{#with movie}}
+
+<form action="/movie/{{_id}}" method="POST">
```

Comment:
When editing and submitting, the POST results in a 404.
Editing the above line as follows fixes the issue:

```
<form action="/movies/{{_id}}" method="POST">
```